proving/postrs/bridge.go

```diff
@@ -0,0 +1,54 @@
+package postrs
+
+// #cgo LDFLAGS: -L/home/bartosz/workspace/post-rs/target/release-clib -lpost
```

Comment:
You have hardcoded your home directory here 😉 

The Makefile should already set the `build` directory as a lookup path for the library, so if you put it there this should be enough:

```suggestion
// #cgo LDFLAGS: -lpost
```

Reply:
I know 😬 It's already removed.